README.md

```diff
@@ -2,11 +2,30 @@
   <img height=75 src="https://github.com/smartface/sample-clienteling/blob/master/temp/smartface_logo.png" alt="smartface Logo" />
 </h3>
 
-# Clienteling from Smartface
```

Comment:
Why the name has changed?
If an explanation to be added, then update the description of the project or add an additional line for that instead of changing the header.